index.html

```diff
@@ -52,5 +52,149 @@ <h1>Hey there. I'm <br>Victor<br><span>
+                </ul>
+                <button class="see-project" type="button">See Project</button>
+            </div>
+        </div>
+        <div class="portfolio three">
+            <div class="thumbnail">
+            </div>
+            <div class="portfolio-details">
+                <h2>Multi-Post Stories<br>Gain+Glory</h2>
+                <ul class="Tech">
+                    <li>Ruby on rails</li>
+                    <li>css</li>
+                    <li>Javascript</li>
+                    <li>html</li>
+                </ul>
+                <button class="see-project" type="button">See Project</button>
+            </div>
+        </div>
+        <div class="portfolio four">
+            <div class="thumbnail">
+            </div>
+            <div class="portfolio-details">
+                <h2>Multi-Post Stories<br>Gain+Glory</h2>
+                <ul class="Tech">
+                    <li>Ruby on rails</li>
+                    <li>css</li>
+                    <li>Javascript</li>
+                    <li>html</li>
+                </ul>
+                <button class="see-project" type="button">See Project</button>
+            </div>
+        </div>
+        <div class="portfolio five">
+            <div class="thumbnail">
+            </div>
+            <div class="portfolio-details">
+                <h2>Multi-Post Stories<br>Gain+Glory</h2>
+                <ul class="Tech">
+                    <li>Ruby on rails</li>
+                    <li>css</li>
+                    <li>Javascript</li>
+                    <li>html</li>
+                </ul>
+                <button class="see-project" type="button">See Project</button>
+            </div>
+        </div>
+        <div class="portfolio six">
+            <div class="thumbnail">
+            </div>
+            <div class="portfolio-details">
+                <h2>Multi-Post Stories<br>Gain+Glory</h2>
+                <ul class="Tech">
+                    <li>Ruby on rails</li>
+                    <li>css</li>
+                    <li>Javascript</li>
+                    <li>html</li>
+                </ul>
+                <button class="see-project" type="button">See Project</button>
+            </div>
+        </div>
+        </div>
+    </section>
+    <section class="about">
+        <div class="about-summary">
+            <h1 class="about-title">About me</h1>
+            <p> I can help you build a product ,<br>
+                feature or website Look through<br>
+                some of my work and experience! If<br>
+                you like what you see and have a<br>
+                project you need coded,<br> 
+                don’t hestiate to contact me.</p>
+            <button class="resume" type="button">Get My Resume</button>
+        </div>
+        <div class="technologies">
+            <div class="tech languages">
+                <div class="icon"><img src="images/Icons_lang.svg" alt="icon-language"></div>
```

Comment:
- [ ] Your design is to salute 👍🏻 , but I think there are a few things to improve before you go to the next project,
      - fix your background for this section as per the original design
      - all your language images are broken too, please fix them up.
      - 
<img width="391" alt="aboutmy" src="https://user-images.githubusercontent.com/79058364/159113858-9e6b16f9-93b7-43c2-b239-dc5d3e1b6318.png">

- it should be like this ⬇️ 
<img width="239" alt="figmyy" src="https://user-images.githubusercontent.com/79058364/159113879-6c66d009-4e62-400f-9f71-e8b634e5554b.png">

Reply:
Ahn.. Apparently i didnt push the images.  Thanks. Fixed it now.

Reply:
@BenMukebo Apparently I didnt push the images to github. But its fixed now. Kindly preview